idd/Energy+.idd.in

```diff
@@ -98525,8 +98525,6 @@ Output:Table:SummaryReports,
        \memo other tabular reports.  These predefined reports are sensitive to the OutputControl:Table:Style object
        \memo and appear in the same files as the tabular reports.  The entries for this object is a list
        \memo of the predefined reports that should appear in the tabular report output file.
-       \memo There should be as many fields (A) in this object as there are keys in the following (minus
-       \memo AllSummary+AllMonthly+AllSummaryAndMonthly)
```

Comment:
This object, Output:Table:SummaryReports has been incredibly long for a long time (https://github.com/NREL/EnergyPlusArchive/commit/7134906aa552086d7fc76fafde2158efe85fe2ac). Taking a bold step here and reducing it to 25 report names in the IDD instead of 82. It's extensible, so any long objects should run just fine. It might trip up someone using IDF Editor, but it seems time to shorten this.

Reply:
I am fine with that bold step.